### DISCUSSION TITLE
Problem logging in using MS account

### DISCUSSION THREAD

**Discussion by bsd-it** [SafeExamBrowser/seb-mac#458] [Q&A]
Hello everyone

On a student's Macbook after logging in with MS accounts, an error appears: AADSTS165000  (I have also tested on version 3.4)

The configuration file is from the Classtime platform and on other devices it works....

After entering the startup code, we choose MS login, enter the user (our login account opens), enter the password and at this moment a window appears whether our domain is **a trusted domain**. We press continue and the error....

![Image](https://github.com/user-attachments/assets/6fe9ccdc-0f1c-47fc-a4f8-790b4e8e8d26)

Maybe someone knows how to deal with this?
Thanks for all the suggestions

Greetings

**Comment by BBZ-SolothurnGrenchen** [comment]
Hello guys

We have the same problem with one of our students SEB. A student can no longer log in to a session with our Microsoft account. The ‘.seb’ file can be opened without any problems and his login in Classtime works, but as soon as he tries to log in in the SEB, this is done up to the point where you get to our login portal. Then you have to choose whether you want to stay logged in or not, as soon as you select one of the options the login continues to load indefinitely. 

There is maybe a similar issue asking for the same solution. It's not the exact same. I subscribed to this Q&A threat aswell, so I'd get notified when there's a solution to that problem, if that would fix my problem aswell I'll mark this threat as solved aswell.

Here a picture of the login that isn't working, or not proceeding to load:
![SEB_LoginIssue](https://github.com/user-attachments/assets/79509f38-d14d-4e6d-9c9c-5bc3faed76ca)


I have tried several things to fix this error, the device is only used by him for school and nothing else, also I can't find any antivirus, network settings or vpn services that could cause this problem. I have tested versions 3.2-3.4 and got the same result with all of them.

Our current workaround is that he does the tasks and checks on another device (here the login works fine and he can use everything), but this should only serve as a temporary solution. so that he can use his MacBook in the future, I would be glad if you could help us further.

Please do not hesitate to contact me if you have any questions or require further information.

I really do believe the solution could fix both of our problems 👍

with kind regards
BBZ Solothurn-Grenchen

**Reply by danschlet** [reply]
Can you please make sure that your exam is using the modern WebView? This can best be verified in SEB logs (please attach some here or send by email to info at safeexambrowser . org
Also if you are able to pinch-to-zoom the web page (on the trackpad), usually the modern WebView is used.
The classic WebView (WebKit browser engine) is outdated, but SEB for macOS still supports it for compatibility reasons. Maybe your exam system or SEB settings need to be updated to support the modern WebView.
Which other discussion are you referring to? Please put a link to it into your issue reports or discussion posts.

**Reply by BBZ-SolothurnGrenchen** [reply]
Good morning

Thank you for your feedback, unfortunately I could only answer today because the student is only present every Thursday. I referred to this Q&A thread because the person who opened this thread had a similar problem, but I mentioned it in a strange way. We were able to solve the problem, he reset his device in his absence. Now everything is working again and he can use the Safe Exam browser again. 

Thank you again for your quick response and I apologise for the late reply.

With kind regards
BBZ Solothurn-Grenchen

**Comment by danschlet** [comment]
The latest release SEB 3.4.1 for macOS displays a deprecation notice about the classic WebView on the top of a browser window when SEB settings/SEB integration forces SEB to fall back to the classic WebView. That could explain issues with web applications/websites, as the classic WebView is a very outdated version of the WebKit browser engine. If this message is displayed on top of these login windows, then the one who created the SEB integration or settings should make sure to update their integration [according to our documentation](https://safeexambrowser.org/developer/seb-config-key.html) or contact us if they don't know why this happens.

If the deprecation warning isn't displayed with SEB 3.4.1, then try if the affected web application works with Safari. Also note that SEB uses the WebKit version of the installed macOS version, so an old macOS version might cause such issues (we recommend like Apple to use the three latest macOS releases).
